src/mongo/db/storage/wiredtiger/wiredtiger_record_store.cpp

```diff
@@ -220,6 +226,7 @@ void WiredTigerRecordStore::OplogStones::createNewStoneIfNeeded(RecordId lastRec
         return;
     }
 
+    LOG(1) << "create new oplogStone, current stones:" << _stones.size();
```

Comment:
At log level 1, we shouldn't really talk about "oplogStone", as that is not a concept that we expect users to understand, so move this to log level 2.